orion/pkg/tests_integration/class_integration_test.go

```diff
@@ -0,0 +1,493 @@
+package integration_tests
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+	"time"
+)
+
+// Test: Create 3 Classes and GetAll()
+func Test_CreateClasses(t *testing.T) {
+	resetTable(t, domains.TABLE_CLASSES)
+
+	now := time.Now().UTC()
+	class1 := createClass(
+		"program1",
+		"2020_spring",
+		sql.NullString{String: "class1", Valid: true},
+		"program1_2020_spring_class1",
+		"churchill",
+		"3 pm - 5 pm",
+		now,
+		now.Add(time.Hour*24*60),
```

Comment:
Can use variables to define these endDates and simply use those variables instead of having to re-calculate them.